### PR TITLE
refactor(behavior_velocity_planner): update launch and parameter files for plugin

### PR DESCRIPTION
**Pull request by isamu-takagi**
## Description

Update launch and parameter files. See https://github.com/autowarefoundation/autoware.universe/issues/3724 for details.
This PR depends on https://github.com/autowarefoundation/autoware.universe/pull/3811.

## Tests performed

Check autonomous driving in planning simulator.

## Effects on system behavior

None.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by maxime-clem — APPROVED** [review]
Tested with https://github.com/autowarefoundation/autoware.universe/pull/3811 and it works well.

File: autoware_launch/launch/components/tier4_planning_component.launch.xml [hunk 1]
```diff
@@ -45,21 +45,22 @@
...
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/crosswalk.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/detection_area.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/stop_line.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/traffic_light.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/no_stopping_area.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/out_of_lane.param.yaml]"
```
Comment (tkimura4) [inline autoware_launch/launch/components/tier4_planning_component.launch.xml:62]: How about using ```behavior_velocity_config_path``` to omit the description?

**Review by shmpwk — APPROVED** [review]
LGTM

File: autoware_launch/launch/components/tier4_planning_component.launch.xml (hunk 1 above)
Comment (isamu-takagi) [reply autoware_launch/launch/components/tier4_planning_component.launch.xml]: Updated https://github.com/autowarefoundation/autoware_launch/pull/369/commits/441fe2f3ad4884cb582d1e7f39a2b1bf64dbf6ec.

**Comment by mehmetdogru** [comment]
@isamu-takagi it seems there are some missing modules in the PR. Is there a reason for that?

**Comment by isamu-takagi** [comment]
@mehmetdogru Thank you. It seems that the merge did not work well because the no_drivable_lane module was added during this PR. I'll add it. Are there any other missing modules?

**Comment by mehmetdogru** [comment]
Under launch modules only no_drivable_lane.

In planning_component.launch.xml there are a few missing:

virtual_traffic_light
occlusion_spot
run_out
speed_bump
no_drivable_lane

Thank you :pray:

**Comment by isamu-takagi** [comment]
Ah, I see. Always load all module parameters and only change `launch_modules`.

**Pull request merged**